phoenix-core/src/it/java/org/apache/phoenix/end2end/StoreNullsIT.java

```diff
@@ -76,6 +81,15 @@
     private final String ddlFormat;
     
     private String dataTableName;
+
+    // In this class we depend on the major compaction to be done instantly
+    // so wo are overwriting the PHOENIX_MAX_LOOKBACK_AGE_CONF_KEY to 0
+    @BeforeClass
```

Comment:
I would expect the change in TestUtil to fix this case as well. 
Why do we need to override PHOENIX_MAX_LOOKBACK_AGE_CONF_KEY here ?

Reply:
My change in TestUtil solves the hanging tests, because there we waited until a put and delete is deleted, which never happened because they were just done before calling the major compaction with a PHOENIX_MAX_LOOKBACK_AGE of an hour.
In StoreNullsIT in testDeletes, we upsert a line into a table, delete it run the major compaction.
Then open a historical connection after the upsert but before the delete, previously when PHOENIX_MAX_LOOKBACK_AGE_CONF_KEY was set to 0 we could not read back the data, but if it is set to an hour, the major compaction will not delete that. So the test would fail.

Reply:
But the comment here is not accurate, thats right.